plugin/lighthouse/setup_test.go

```diff
@@ -2,23 +2,87 @@ package lighthouse
 
 import (
 	"github.com/caddyserver/caddy"
+	"github.com/coredns/coredns/plugin/pkg/fall"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("[setup] Test basic bring up", func() {
+var _ = Describe("[setup] Test lighthouse bring up", func() {
+	Context("When configuration is correct", testCorrectConfig)
+	Context("When configuration is incorrect", testIncorrectConfig)
+})
 
+func testCorrectConfig() {
 	It("Should come up without errors", func() {
 		c := caddy.NewTestController("dns", `lighthouse`)
-		err := setupLighthouse(c)
+		lh, err := lighthouseParse(c)
 		Expect(err).NotTo(HaveOccurred())
+		Expect(lh.Clusters).Should(BeEmpty())
+		Expect(lh.Fall).Should(Equal(fall.F{Zones: nil}))
+		Expect(lh.Zones).Should(BeEmpty())
+		Expect(lh.Clusters).Should(BeEmpty())
 	})
 
-	It("Should give error on setup", func() {
-		c := caddy.NewTestController("dns", `lighthouse more`)
-		err := setupLighthouse(c)
-		Expect(err).To(HaveOccurred())
+	It("Should populate fields correctly", func() {
+		config := `lighthouse cluster2.local cluster3.local {
+			clusters cluster2 cluster3
+            fallthrough cluster2.local
```

Comment:
Indentation looks off here...

Reply:
Looked correct in my IDE. tab vs spaces thing. Are we supposed to use tab or spaces?